contracts/polls/PollManager.sol

```diff
@@ -317,8 +321,46 @@ contract PollManager is Controlled {
         return votes;
     }
 
+    /// @notice Claim rewards for voting in poll
+    /// @param _idPoll Poll
+    function claimReward(uint _idPoll)
+      public
+    {
+      require(_idPoll < _polls.length, "Invalid _idPoll");
+      Poll storage p = _polls[_idPoll];
+      require(block.number >= p.startBlock && block.timestamp < p.endTime && !p.canceled, "Poll is inactive");
```

Comment:
Can rewards be claimed only while the poll is active?

Reply:
No, good catch.